packages/api-contracts/ai-scan-api.yaml

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+openapi: 3.0.0
+info:
+    version: 1.0.0
+    title: Accessibility Insight Service – Scan API
+paths:
+    /scans:
+        post:
+            summary: Create accessibility scan requests
+            operationId: createScans
+            parameters:
+                - name: api-version
+                  in: path
+                  required: true
+                  description: The client API version
+                  schema:
+                      type: string
+                  example: '1.0'
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ScanRunRequests'
+            responses:
+                '201':
```

Comment:
Should we consider using HTTP 202 instead of HTTP 201?
202 indicates that the request was accepted but a resource was not created. [More Info](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)

Reply:
Updated to 202